tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -315,7 +322,9 @@ class VarDictJavaEndToEnd
     Io.assertCanWriteFile(out)
     normalBam.foreach(Io.assertReadable)
 
-    val tName = tumorName.getOrElse(VarDictJava.extractSampleName(bam = tumorBam))
+    val tName = tumorName
```

Comment:
```suggestion
    val _tomurName = tumorName
```
convention is to use undercase if you want to re-use the same variable name.  `tName` is not a good name.